x-pack/legacy/plugins/siem/public/components/formatted_bytes/index.tsx

```diff
@@ -12,9 +12,7 @@ import { useUiSetting$ } from '../../lib/kibana';
 
 export const PreferenceFormattedBytesComponent = ({ value }: { value: string | number }) => {
   const [bytesFormat] = useUiSetting$<string>(DEFAULT_BYTES_FORMAT);
-  return (
-    <>{bytesFormat ? numeral(value).format(bytesFormat) : numeral(value).format('0,0.[0]b')}</>
-  );
+  return <>{numeral(value).format(bytesFormat || '0,0.[0]b')}</>;
```

Comment:
nit, can we create a `const` for '0,0.[0]b' since we are also using it there https://github.com/elastic/kibana/pull/54030/files#diff-5839ee2c63a46315927b26809c783ec7R22

Reply:
@XavierM I think we're actually going to remove this particular usage. I just noticed we're using it [here](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/public/components/matrix_histogram/utils.ts#L92) as well, which I'm going to inline to pull from UI settings as well. (cc @angorayc)

That just leaves references in the tests. I'm happy to declare a `const DEFAULT_BYTES_FORMAT_VALUE = '0,0.[0]b'` at the top of the test file for clarity. Does that work?

Reply:
@XavierM @angorayc I refactored the byte formatting for our DNS Histogram to use the UI Setting rather than a hardcoded format: e071875. I don't have data populating in that chart, so I'm not able to test it out locally, but let me know if there are any issues there.